frontend/src/api/Room.ts

```diff
@@ -82,7 +82,11 @@ export const changeRoomHost = (roomId: string, roomParams: ChangeHostParams):
   });
 
 export const removeUser = (roomId: string, roomParams: RemoveUserParams):
-  Promise<Room> => axios.put<Room>(routes.removeUser(roomId), roomParams)
+  Promise<Room> => axios({
+  url: routes.removeUser(roomId),
+  method: 'DELETE',
+  data: roomParams,
+})
```

Comment:
Is there a reason you didn't use `axios.delete`. [[1]](https://stackoverflow.com/questions/51069552/axios-delete-request-with-body-and-headers) If we are switching it to this format, should we keep it consistent across the rest of the `axios` functions?

Reply:
This probably has something to do with the issue I couldn't figure out where you'll keep getting Bad Request when you try to send a body with the delete request without enclosing it in "data".

Reply:
Yeah I think I ended up coming across the same error as Helen, even when using `config.data` with `axios.delete`for the request body as a ton of posts online suggested, so this ended up being the first solution for me that worked. Personally I think this is less clean so I'm ok with leaving the other ones as they are, unless you have any preference for consistency sake or like seeing if there are other ways to get `axios.delete` to work

Reply:
I'll merge for now but am free to update this anytime